src/SDKs/Storage/Management.Storage/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "6.2.0-preview",
+  "version": "6.3.0-preview",
```

Comment:
Delete this file. This file is no longer needed.
Also delete .xproj and global.json (if they are still in the directory)